### ISSUE TITLE
Wrong definition of gaussopt.BeamParameter.radius

### ISSUE THREAD

**Issue by operte** [sympy/sympy#9707]
Hi everyone,

I've been using the gaussopt module (Sympy 0.7.6) a lot recently but I have been having problems with calculating the Radius of Curvature (RoC) of a gaussian beam correctly. Today I thought that there could be some problem with the definition of the RoC in the module and it appears to me that this is the case. 

In sympy, the RoC is defined as follows:
```python
    def radius(self):
        """
        The radius of curvature of the phase front.

        Examples
        ========

        >>> from sympy.physics.optics import BeamParameter
        >>> p = BeamParameter(530e-9, 1, w=1e-3)
        >>> p.radius
        0.2809/pi**2 + 1
        """
        return self.z*(1 + (self.z/self.z_r)**2)
```
http://docs.sympy.org/dev/_modules/sympy/physics/optics/gaussopt.html#BeamParameter.radius

However, the definition is `z_r/z`, and not `z/z_r`:
https://en.wikipedia.org/wiki/Gaussian_beam#Radius_of_curvature

Could someone please verify this situation?

**Comment by jksuom** [comment]
I think you are right. The expression in parentheses seems to have been mistakenly copied from ``w``. Please send a PR for the fix.

**Comment by operte** [comment]
I'm sorry but I'm new to git. Could you please teach me how to properly submit a PR?

**Comment by jksuom** [comment]
[Here](https://github.com/sympy/sympy/wiki/Development-workflow) you will find a good starting point.

**Comment by jksuom** [comment]
Fixed by #9875
